torch/_inductor/codegen/triton.py

```diff
@@ -84,10 +84,6 @@ def _print_floor(self, expr):
         assert len(expr.args) == 1
         return f"tl.libdevice.floor({self.paren(self._print(expr.args[0]))})"
 
-    def _print_ceiling(self, expr):
-        assert len(expr.args) == 1
-        return f"tl.libdevice.ceil({self.paren(self._print(expr.args[0]))})"
```

Comment:
Can I suggest that we instead raise an error when this happens with instructions on how to escalate when someone attempts to print ceil?

Reply:
I'll do that when I get to the end of pnasnet, so far it's more convenient to have fully generated source that errors out during compilation, rather than error out during codegen.

Reply:
Another possibility is keep this line but codegen it to ceiling_THISSHOULDNOTBEHERECHECKISSUE27384